docs/source/index.rst

```diff
@@ -83,7 +83,7 @@ Testimonials
 
 
 .. _Django REST Framework: https://www.django-rest-framework.org/
-.. _Two Scoops of Django: https://www.twoscoopspress.com/products/two-scoops-of-django-1-11
+.. _Two Scoops of Django: https://www.feldroy.com/two-scoops-press#two-scoops-of-django
```

Comment:
Let's add a little workflow invoking `make linkcheck`?

Reply:
... too late, merged already. ;]

Reply:
@amotl that's a good idea